src/plugin.ts

```diff
@@ -32,6 +32,6 @@ function activateWidgetExtension(
     registry.registerWidget({
         name: MODULE_NAME,
         version: MODULE_VERSION,
-        exports: () => import('./index'),
+        exports: (): any => import('./index'),
```

Comment:
this gets rid of the typescript error, but is probably not the best way to do that.